styles/autocomplete.less

```diff
@@ -59,6 +59,7 @@ autocomplete-suggestion-list.select-list.popover-list {
     margin-top: 0;
     display: table;
     width: 100%;
+    margin-right: 10px; // Needed to prevent horizontal scrolling when right label is too long
```

Comment:
I am on OSX. It's very strange that you aren't seeing the issue. I've configured the `font-size` to be `13px` and I am using the [Hack](https://github.com/chrissimpkins/Hack) font, but other than that my styles are pretty standard. I am using the `Atom Dark` theme. Here's two recordings to show the issue:

## Without `margin-right: 10px` (bad horizontal scroll bar)

![without-margin-00](https://cloud.githubusercontent.com/assets/978214/15969508/9eb9ea04-2eed-11e6-8bdd-ba392d3efd58.gif)


## With `margin-right: 10px` (no horizontal scroll bar)

![with-margin](https://cloud.githubusercontent.com/assets/978214/15969449/62918a78-2eed-11e6-9c73-57946466863a.gif)